python/private/py_executable.bzl

```
@@ -786,7 +786,7 @@ def _create_stage1_bootstrap(
         )
         template = runtime.bootstrap_template
         subs["%shebang%"] = runtime.stub_shebang
-    else:
+    elif main_py:
```

```suggestion
    elif not ctx.files.srcs:
        fail("mandatory 'srcs' files have not been provided")
    else:
```
It is probably better to do it this way instead of failing at the very end.